pages/data-providers/[data-provider-id]/deposit-compliance.jsx

```diff
@@ -40,33 +40,50 @@ const SidebarContent = ({ context: { oai, originalId, authors, title } }) => {
  *       of Layout.Main and explicitly pass props.
  *       We should get rid out of it!
  */
-const DepositDates = ({
+const DepositCompliance = ({
   className,
   store,
   tag: Tag = 'main',
   ...restProps
 }) => (
   <Tag className={[styles.container, className].join(' ')} {...restProps}>
     <h1>Deposit compliance</h1>
+    {store.depositDates.complianceLevel && (
+      <Markdown>
+        {texts.compliance.body.render({
+          amount: (100 - store.depositDates.complianceLevel[1]).toFixed(1),
+        })}
+      </Markdown>
+    )}
 
-    <Card className={styles.complianceLevel} tag="section">
-      <h2>{texts.compliance.title}</h2>
-      {store.depositDates.complianceLevel ? (
-        <Markdown>
-          {texts.compliance.body.render({
-            amount: (100 - store.depositDates.complianceLevel).toFixed(1),
-          })}
-        </Markdown>
-      ) : (
-        <p>{texts.noData.body}</p>
-      )}
+    <Card className={styles.dataOverview} tag="section">
+      <h2>{texts.dataOverview.title}</h2>
+      <div className={styles.numbers}>
+        <NumericValue
+          tag="div"
```

Comment:
I still would use `p` here as it represents a self-contained piece of information that sounds exactly like _2,043 outputs counted_, it's a sentence.